tools/TerminalVelocity/Generate-TerminalVelocityFeatures.ps1

```diff
@@ -46,6 +46,8 @@ Param(
     [string]$Namespace
 )
 
+Set-StrictMode -Version 3.0
```

Comment:
Why not also `$ErrorActionPreference = 'Stop'`

Reply:
Because it already exists on line #177.